src/components/common/Modal.tsx

```diff
@@ -6,13 +6,15 @@ export const Modal = ({
   onClose,
   escapeToClose = true,
   clickOutsideModalToClose = false,
+  unstyled = false,
   bgColor = '#FAFAFA',
   children,
 }: {
   isOpen: boolean;
   onClose: () => void;
   escapeToClose?: boolean;
   clickOutsideModalToClose?: boolean;
+  unstyled?: boolean;
```

Comment:
Do we need this? Right now, this modal is only used for the iFrame for `PayWithCard` and `CheckoutWithCard` when showing KYC modal

If we think that this should be the default behavior, we can remove those styling // close button.